packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -52,8 +48,6 @@ BuildRequires: rubygem(gettext_i18n_rails)
 BuildRequires: rubygem(apipie-rails) >= 0.5.14
 BuildRequires: rubygem(fx) < 1.0
 BuildRequires: rubygem(pg)
-BuildRequires: rubygem(runcible) >= 2.13.0
```

Comment:
We can finally drop this, let's not forget the follow up PR to remove it.

Reply:
https://github.com/theforeman/foreman-packaging/pull/9321